jetty9-compat-base/src/test/java/com/google/apphosting/vmruntime/jetty9/JettyRunner.java

```diff
@@ -255,11 +267,61 @@ public void run() {
   }
 
   /**
+   * Find the log file that contains the specific content pattern.
+   * <p>
+   * This exists to get around the fact that the java.util.logging
+   * layer will generate a new log file each time its reset/initialized
+   * because it sees the lock from the previous instance before the
+   * previous instance has a chance to close and remove the lock.
+   * </p>
+   * <p>
+   * This function will iterate through all of the log files and
+   * attempt to find the specific pattern you are interested in.
+   * </p>
+   *
+   * @param regex the text (regex) pattern to look for in the log file.
+   * @return the File for the discovered log file
+   * @throws java.io.FileNotFoundException if unable to find a log
+   * file with the specific text pattern
+   * @throws IOException if unable to read a log file and/or pattern
+   */
+  public File findLogFileThatContains(String regex) throws IOException {
```

Comment:
This class is not a test but rather a test helper.
Currently (I think) the only test that is actually using this method is `LoggingIT` which is where I was suggesting to move it to.